tensorflow/core/platform/hadoop/hadoop_file_system.h

```diff
@@ -77,7 +77,8 @@ class HadoopFileSystem : public FileSystem {
   string TranslateName(const string& name) const override;
 
  private:
-  std::map<std::string, hdfsFS> connectionCache_;
+  mutable mutex mu_;
```

Comment:
I think `mutable` isn't necessary here 😕 but I am not sure.

You will have to include the header
```cpp
#include "tensorflow/core/platform/mutex.h"
```

and to BUILD
```
deps = [..., "//tensorflow/core/platform:mutex" ]
```

You can run `buildifier` locally to pass `Ubuntu Sanity`.

Reply:
I refer to [this](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/platform/hadoop/hadoop_file_system.cc#L295), but I also don't think it is necessary to add `mutable` here. I will remove it.